test/unit/builtins/numbers.spec.ts

```diff
@@ -69,6 +69,13 @@ test.each([
     util.testExpressionTemplate`(${value}).toString(2)`.expectToEqual(luaNativeSpecialNumString);
 });
 
+const toFixedRadixes = [undefined, 0, 1, 2, 5, 10, 80];
+const toFixedValues = [-1, 0, 1, /* 1.5 fails, */ 1024, 1.2, 1e19, 1e22];
```

Comment:
Please add a value with 6+ decimal places here

Reply:
Added a number and Math.PI